src/ripple/app/tx/impl/CreateOffer.cpp

```diff
@@ -239,13 +239,13 @@ CreateOffer::bridged_cross (
         throw std::logic_error ("Bridging with XRP and an endpoint.");
 
     OfferStream offers_direct (view, view_cancel,
-        Book (taker.issue_in (), taker.issue_out ()), when, ctx_.config, j_);
+        Book (taker.issue_in (), taker.issue_out ()), when, j_);
```

Comment:
We use `j` and `j_` to mean the relevant instance of `beast::Journal` since it occurs so frequently